cluster/node-pools/master-default/userdata.yaml

```diff
@@ -664,7 +664,11 @@ write_files:
         hostNetwork: true
         containers:
         - name: kube-scheduler
-          image: nonexistent.zalan.do/teapot/{{if eq .Cluster.ConfigItems.kubernetes_scheduler_image "zalando" }}kube-scheduler-internal{{else}}kube-scheduler{{end}}:fixed
+{{- if eq .Cluster.ConfigItems.kubernetes_scheduler_image "zalando" }}
+          image: 926694233939.dkr.ecr.eu-central-1.amazonaws.com/production_namespace/teapot/kube-scheduler-internal:v1.30.1-master-121
+{{- else }}
+          image: nonexistent.zalan.do/teapot/kube-scheduler:fixed
+{{- end }}
```

Comment:
Do we want to include https://github.com/zalando-incubator/kubernetes-on-aws/pull/7735 here?

Reply:
I would rather merge this, so we can start the upgrade today, otherwise we have to wait at least a day and get closer load test and then have to wait even longer. We ran this (unintentionally) for the past weeks successfully, so from a safety perspective we're good IMO.